hr_salary_rule_reference/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=====================
+Salary Rule Reference
+=====================
+
+Add a reference to the salary rule itself in its local context
+
+The rule can be referenced with its id with 'rule_id'
+or a browse record with 'rule'.
+
+The reference to the payslip inside the salary rule is now a record set from the v8 api.
+
+Installation
+============
+
+There is no specific installation instruction for this module.
+
```

Comment:
Kindly add usage section so it will help end user to check on runbot.
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

Reply:
Thanks, I added Usage as the new template of OCA guidelines.